e2e/cypress/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace Cypress {
+  interface Chainable<Subject> {
+    login(url: string, username: string): Cypress.Chainable<any>
```

Comment:
❓ Can we do better than `any` here? [Maybe `{}`?](https://mariusschulz.com/blog/typescript-2-2-the-object-type#object-vs-object-vs)